code/game/machinery/iv_drip.dm

```diff
@@ -400,13 +400,15 @@
 
 ///modified IV that can be anchored and takes plumbing in- and output
 /obj/machinery/iv_drip/plumbing
-	name = "automated IV drip"
+	name = "Automated IV Drip"
```

Comment:
Don't change any of these

Reply:
I need to capatilze the name so it looks consistent on the window.  I think I'll do that in the ui_data() function then. What's the function to capatilze the first letter of a string?

Reply:
`capitalize` in string.js to capitalize first letter, `capitalizeAll` in string.js to capitalize all words